### ISSUE TITLE
ReportConverter should not create separate test case for each action

### ISSUE THREAD

**Issue by metaph3r** [MicroFocus/ADM-FT-ToolsLauncher#78]
**Tool**
ReportConverter 1.0.21.1241

**Is your feature request related to a problem? Please describe.**
It seems that the ReportConverter create a separate JUnit test case für each action in a UFT One test case. That is not ideal because if you push the results to Octane with the test result collection tool you end up with very many automated tests which just belong to one UFT One test case.

I do not really understand the intention behind splitting each action into a separate test case and how this should then be managed in the CI server of Octane.

**Describe the solution you'd like**
Each UFT One test case should only create 1 JUnit test case.

**Describe alternatives you've considered**
none

**Environment and testing tools (please complete the following information):**
 - UFT One version: 2021
 - ALM Octane version: 16.0.200

**Additional context**
![image](https://user-images.githubusercontent.com/31897953/155111973-4667ef41-fd58-4d5e-ad12-78e882ad0059.png)
![image](https://user-images.githubusercontent.com/31897953/155112035-eae4635e-169d-4e58-b646-9ef1bbc70090.png)
[result.txt](https://github.com/MicroFocus/ADM-FT-ToolsLauncher/files/8115746/result.txt)

**Comment by soleverxu** [comment]
Hi, thank you for reporting the issue. For the details of the mappings between the UFT One result xml and Junit xml, you can refer to [this wiki page](https://github.com/MicroFocus/ADM-FT-ToolsLauncher/wiki/ReportConverter-Technical-Details#conversion-map---junit).

I don't quite understand what you suggest. Could you please describe the idea with more details? What do you mean `UFT One test case` here?
> **Describe the solution you'd like**
> Each UFT One test case should only create 1 JUnit test case.

**Comment by metaph3r** [comment]
The wiki page is confirming my concern. Why is a UFT One GUITest mapped to a JUnit test suite and each step to a test case? By this approach you end up with the issue in my first screenshot if you transfer the test results to Octane. You have an automated test for each test step.

The FTToolsLauncher does only create 1 JUnit test case per UFT One GUITest and I would expect the same from the report converter since it belongs to the same tool suite.

To outline our use case. We are running UFT tests in a Bamboo Job using the Octane Plugin. But we also want to make the test results available to another tool that support the JUnit format. So our idea is to use the Report Converter to create the JUnit format. But right now it is not feasibly because ever UFT step results in a sperate test case.

**Comment by soleverxu** [comment]
So, if I'm not misunderstood, what you want is a Junit Xml file converted from the UFT One test result with the following mapping:
`UFT One test result` <-> `Junit testcase`
, right?

**Comment by metaph3r** [comment]
> UFT One test result <-> Junit testcase

Exactly. Having an automated tests für each UFT One Action is very confusing.
